src/main/java/com/powsybl/openloadflow/ac/PhaseControlOuterLoop.java

```diff
@@ -97,14 +96,16 @@ public OuterLoopStatus check(OuterLoopContext context, Reporter reporter) {
     private OuterLoopStatus firstIteration(OuterLoopContext context) {
 
         List<DiscretePhaseControl> phaseControlsOn = context.getNetwork().getBranches().stream()
-            .map(branch -> branch.getDiscretePhaseControl().filter(dpc -> branch.isPhaseControlled()))
-            .filter(Optional::isPresent)
-            .map(Optional::get)
-            .filter(dpc -> dpc.getMode() == DiscretePhaseControl.Mode.CONTROLLER || dpc.getMode() == DiscretePhaseControl.Mode.LIMITER)
+            .filter(branch -> branch.isPhaseControlled()
+            /*        && branch.isPhaseControlEnabled() FIXME */
```

Comment:
@annetill it seems to broke some more unit test if I check branch phase control is on before switching control!

Reply:
What is meant by branch control is on before switching control

Reply:
I am checked some tests and it seems that even if you save a property, the properties are null.